src/stories/5-custom/array-field/array-field.stories.tsx

```diff
@@ -330,6 +332,11 @@ const CustomErrorStory = <T,>(id: string, showSubmitButton = true) =>
 											name: "Nested custom error",
 											color: "Nested custom error",
 										},
+										undefined,
```

Comment:
in order to be clearer, can we follow the structure indicated in the ACs? so it will contain name, email, uinfin, the nested array-field will contain child name and child name

customise the add button label too, main button should be "Add member" and nested button should be "Add child"

also see if you can wrap the nested array-field with a border to differentiate from the main array-field